ingest/README.md

```diff
@@ -31,7 +31,7 @@ This will produce two files (within the `ingest` directory):
 Run the complete ingest pipeline and upload results to AWS S3 with
 
 ```sh
-nextstrain build . --configfiles defaults/config.yaml defaults/optional.yaml
+nextstrain build . --configfiles defaults/config.yaml build-defaults/nextstrain-automation/config.yaml
```

Comment:
FYI, since the defaults/config.yaml is being used by default in the [Snakefile](https://github.com/nextstrain/zika/blob/1654ca4238959b60498afc955b70f9620d0b0a60/ingest/Snakefile#L8), you can drop the first config file in the example command. 

```suggestion
nextstrain build . --configfiles build-defaults/nextstrain-automation/config.yaml
```